### PR TITLE
SNOW-897060 - Release 1.5.0

### PR DESCRIPTION
**Pull request by sfc-gh-anavalos**
Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

**Review by sfc-gh-sfan — APPROVED** [review]

**Comment by sfc-gh-aling** [comment]
the version in https://github.com/snowflakedb/snowflake-sqlalchemy/blob/main/src/snowflake/sqlalchemy/version.py also needs to be updated

**Review by sfc-gh-aling — APPROVED** [review]

File: DESCRIPTION.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Source code is also available at:
 
 # Release Notes
 
-- v1.4.8(TBD)
+- v1.4.8(Aug 23, 2023)
```
Comment (sfc-gh-aling) [inline DESCRIPTION.md:12]:
```suggestion
- v1.5.0(Aug 23, 2023)
```

File: DESCRIPTION.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Source code is also available at:
...
 
   - Added option to create a temporary stage command.
   - Add support for geometry type.
```
Comment (sfc-gh-aling) [inline DESCRIPTION.md:15]:
```suggestion
  - Added support for geometry type.
```

File: src/snowflake/sqlalchemy/version.py [hunk 3]
```diff
@@ -3,4 +3,4 @@
 #
 # Update this for the versions
 # Don't change the forth version number from None
-VERSION = (1, 4, 7, None)
+VERSION = (1, 4, 8, None)
```
Comment (sfc-gh-aling) [inline src/snowflake/sqlalchemy/version.py:6]:
```suggestion
VERSION = (1, 5, 0, None)
```

Comment (sfc-gh-anavalos) [reply src/snowflake/sqlalchemy/version.py]: Done

File: DESCRIPTION.md (hunk 2 above)
Comment (sfc-gh-anavalos) [reply DESCRIPTION.md]: Done

File: DESCRIPTION.md (hunk 1 above)
Comment (sfc-gh-anavalos) [reply DESCRIPTION.md]: Done

**Pull request merged**